main.py

```diff
@@ -1,66 +1,89 @@
+# coding=utf-8
+
 import sys
 import os
 from subprocess import call
 import argparse
 
-VALID_IMAGES = [".jpg",".gif",".png",".tga",".tif",".bmp"]
+VALID_IMAGES = [".jpg", ".gif", ".png", ".tga", ".tif", ".bmp"]
 FNULL = open(os.devnull, 'w')
 
+
 class ArgumentMissingException(Exception):
-	def __init__(self):
-		print("usage: {} <dirname>".format(sys.argv[0]))
-		sys.exit(1)
+    def __init__(self):
+        print("usage: {} ".format(sys.argv[0]))
+        sys.exit(1)
+
+
+def create_directory(dir_path):
+    if not os.path.exists(dir_path):
+
+        os.makedirs(dir_path)
 
-def create_directory(path):
-	if not os.path.exists(path):
-	    os.makedirs(path)
 
-def check_path(path):
-	return bool(os.path.exists(path))
+def check_path(dir_path):
+    return bool(os.path.exists(dir_path))
 
-def main(input_path, output_path):
-	if call(['which', 'tesseract']):
-		print("tesseract-ocr missing, use sudo apt-get install tesseract-ocr to resolve")
-	elif check_path(input_path):
 
-		count = 0
-		other_files = 0
+def main(source, output_dir):
+    current_directory = os.getcwd()
+    destination = output_dir
+    if not check_path(destination):
+        destination = current_directory + output_dir
+        create_directory(destination)
+        if not check_path(destination):
+            print(
+                "I could not create or find '%s'" % destination
```

Comment:
Do not use message like this, use something like "Directory could not be created at %s"

Reply:
Why?